main/src/modules/reactiveLayer/behaviorManager/src/behaviorManager.cpp

```diff
@@ -75,10 +75,10 @@ bool BehaviorManager::configure(yarp::os::ResourceFinder &rf)
         Time::delay(1.0);
     }
 
-    while (!Network::connect("/ears/behavior:o", rpc_in_port.getName())) {
+    /*while (!Network::connect("/ears/behavior:o", rpc_in_port.getName())) {
         yWarning() << "Ears is not reachable";
         yarp::os::Time::delay(0.5);
-    }
+    }*/
```

Comment:
Finally we get rid of this line. Very good! However, where is the connection gone? Is it not needed anymore? Or should it be added to the WYSIWYD-Year2-XML file?